umap/static/umap/js/umap.js

```diff
@@ -90,6 +90,7 @@ L.U.Map.include({
 
     // After calling parent initialize, as we are doing initCenter our-selves
     if (geojson.geometry) this.options.center = this.latLng(geojson.geometry)
+    this.urls = new window.URLs(this.options.urls)
```

Comment:
Where does the `options` object comes from and how can we ensure the `urls` property exists?
Same question for other properties of `options`